src/plone/restapi/deserializer/dxcontent.py

```diff
@@ -62,6 +62,22 @@ def __call__(self, validate_all=False, data=None):  # noqa: ignore=C901
                     if not self.check_permission(write_permissions.get(name)):
                         continue
 
+                    # set the field to missing_value if we receive null
+                    if data[name] is None:
+                        if not field.required:
+                            dm.set(field.missing_value)
+                        else:
+                            errors.append({
+                                'field': field.__name__,
+                                'message': (
+                                    '{} is a required field.'.format(
+                                        field.__name__
+                                    ),
+                                    'Setting it to null is not allowed.'
+                                ),
+                                'error': None})
```

Comment:
Hmm, `'error': None` feels a bit clunky. I don't have a clear suggestion in mind though.